src/Michał/Fsd.Michal.Cs.Ex1/Fsd.Michal.Cs.Ex1/Program.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+
+
+namespace Fsd.Michal.Cs.Ex1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Header();
+
+            if (args.Length > 0)
+            {
+                for (int i = 0; i < args.Length; i++)
+                {
+                    Console.WriteLine($"Argument { i + 1}  of {args.Length}");
+                    Console.WriteLine(($"Value : {args[i]}"));
+                    Console.WriteLine($"Upper letters: {WordAnalyzer.CountUpperLetters(args[i])}");
+                    Console.WriteLine($"Lower letters: {WordAnalyzer.CountLowerLetter(args[i])}");
+                    Console.WriteLine($"Digits: {WordAnalyzer.CountDigits(args[i])}\n");
+                }
+            }
+
```

Comment:
pomiędzy if {} else {} nie robi się odstępu 
jak if ma > 1 linie to jest dobrą praktyką że są klamry zarówno w if jak i w else